aries-site/src/pages/components/card.mdx

```diff
@@ -51,9 +57,14 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- When presenting high-level, summarized pieces of information.
+- When presenting an item or collection of items that can best be consumed visually.
```

Comment:
- When presenting an item or collection of alike items that can best be consumed visually.

Reply:
I had refrained from saying "alike" since call-to-action cards can be independent

Reply:
Ah yes, indeed. Ignore :)